malformation in @polkadot/api/types.d.ts

I got this error in a project when i tried to upgrade `@pokadot/api` to v0.82.0-beta.96, and i think it also exists in 0.90.0-beta.x
```
../node_modules/@polkadot/api/types.d.ts:1:23 - error TS2688: Cannot find type definition file for 'filesystem'.

1 /// <reference types="filesystem" />
                        ~~~~~~~~~~

../node_modules/@polkadot/api/types.d.ts:135:22 - error TS2304: Cannot find name 'Metadata'.

135     runtimeMetadata: Metadata;

```

check https://unpkg.com/@polkadot/api@0.90.0-beta.14/types.d.ts

--- a/packages/api/src/types.ts
+++ b/packages/api/src/types.ts
@@ -11,7 +11,7 @@ import { DeriveCustom } from '@polkadot/api-derive';
 import { Constants } from '@polkadot/api-metadata/consts/types';
 import { RpcInterface } from '@polkadot/rpc-core/jsonrpc.types';
 import { ProviderInterface, ProviderInterfaceEmitted } from '@polkadot/rpc-provider/types';
-import { u64 } from '@polkadot/types';
+import { Metadata, u64 } from '@polkadot/types';
 import { StorageEntry } from '@polkadot/types/primitive/StorageKey';
 
 import ApiBase from './Base';

